src/app/donate/new-donate/page.tsx

```diff
@@ -1,22 +1,27 @@
 import CampaignCard from "@/components/donate/CampaignCard";
+import { Container, Grid } from "@radix-ui/themes";
 import { FUND_RAISERS } from "@/data/donate";
 
 const Page = () => {
   return (
-    <div>
-      {FUND_RAISERS.map((donate, i) => (
-        <CampaignCard
-          key={i}
-          imgSrc={donate.img}
-          imgAlt=""
-          title={donate.title}
-          raised={donate.percentage}
-          goal={donate.goal_price}
-          donateLink={donate.donate_link}
-          moreLink={donate.more_link}
-        />
-      ))}
-    </div>
+    <Container px="4">
+      <Grid gap="8" columns={{ xs: "2", md: "3" }} pt="4.5rem" pb="9rem">
+        {FUND_RAISERS &&
+          FUND_RAISERS.length > 0 &&
```

Comment:
**suggestion (non-blocking):** Recommend moving the `FUND_RAISERS && FUND_RAISERS.length > 0` checks to outside the container, so that we don't show an empty section if there are no fundraisers.

This makes more sense in context of the full page with the static header / general donate links.  That's obviously not available yet (another PR pending for it) but we can go ahead and make this change now if you want.